packages/next/src/server/lib/cache-handlers/default.ts

```diff
@@ -61,41 +65,30 @@ export const DefaultCacheHandler: CacheHandler = {
     }
   },
 
-  set: async function set(cacheKey, entry) {
+  async set(cacheKey, pendingEntry) {
     let resolvePending: () => void = () => {}
     const pendingPromise = new Promise<void>((resolve) => {
       resolvePending = resolve
     })
     pendingSets.set(cacheKey, pendingPromise)
 
-    const timestamp = performance.timeOrigin + performance.now()
```

Comment:
I think it's important that we use the original time stamp that's passed to us that was gathered before we started gathering the data, because otherwise if there was some revalidate timestamped between when we first start rendering and now, we'd not drop this even though it could've affected this entry.